src/linpred.jl

```diff
@@ -227,13 +227,11 @@ function show(io::IO, obj::LinPredModel)
     println(io, "$(typeof(obj)):\n\nCoefficients:\n", coeftable(obj))
 end
 
-modelframe(obj::LinPredModel) = obj.fr
```

Comment:
why is this being removed?

Reply:
I didn't find any `LinPredModel` here that would have this attribute. There's `LinearModel` which doesn't and through `AbstractGLM` there's `GeneralizedLinearModel` which also doesn't have `fr`.